ocaml-lsp-server/src/code_actions/action_destruct.ml

```diff
@@ -34,9 +36,15 @@ let code_action doc (params : CodeActionParams.t) =
       let finish = Position.logical params.range.end_ in
       Query_protocol.Case_analysis (start, finish)
     in
-    let+ res = Document.dispatch doc command in
+    let* res = Document.dispatch doc command in
     match res with
-    | Ok res -> Some (code_action_of_case_analysis doc uri res)
+    | Ok (loc, newText) -> (
+      let+ formattedText =
+        Ocamlformat_rpc.(format_type state.ocamlformat_rpc ~typ:newText)
```

Comment:
Why do you need the module open? I think `Ocamlformat_rpc.format_type` should be enough.

Reply:
You are right, it is enough. I don't remember why I opened it.